### ISSUE TITLE
Print differently when some unfocused goals remain.

### ISSUE THREAD

**Issue by Zimmi48** [EugeneLoy/coq_jupyter#40]
Currently, if I use focusing brackets `{` or bullets `-` to prove some subgoal, when I'm finished the notebook prints:

```
No more subgoals
```

which is the same as if no subgoals remained at all. Jupyter should hint that there are some unfocused goals remaining like CoqIDE does:

```
This subproof is complete, but there are some unfocused goals:
```

**Comment by EugeneLoy** [comment]
@Zimmi48 Hi. Thanks, I'll look into it.

**Comment by EugeneLoy** [comment]
@Zimmi48

Kernel now outputs focused\unfocused goals similar to CoqIDE.

This is included in `v1.5.3`.

BTW, let me know if you have any other issues\comments based on using kernel in your class.

**Comment by Zimmi48** [comment]
Will do, thanks!
